### ISSUE TITLE
Analytické vrstvy pro experty

### ISSUE THREAD

**Issue by vop-cz** [osmcz/osmcz#52]
Hodilo by se mít možnost zapínat vrstvy zobrazující známé chyby. Konkrétních řešení jsem viděl v průběhu času už více - od nastavitelných (do dialogu se zadá nějaká logická definice, např. way je otagovana jako ulice a soucasne nema name apod.) az po vysoce specializovane, (napr. k opening hours - je to objekt s typickou vlastnosti pravidelne otviraci doby a nema opening hours definovane).

Podobne by si zaslouzil specializovanou vrstvu i projekt rozcestniku - uz se o tom nekde mluvilo - u ikony rozcestniku by mela byt graficky informace, ze k nemu neni zadna fotka, nebo ze k nemu neni ref. Stacily by treba jen nejake barevne pixely, nebo vyuzit tu ikonu rozcestniku a obarvit mu sipecky pro jedntlive sledovane pripady... Uzivatel, ktery vi, co hleda, celkem rychle zjisti, ze ten konkretni dany rozcestnik je chudinka nedokonaly a vi v cem je nedokonaly...

**Comment by mkyral** [comment]
Jo. Tyhle extra věci pro mappery by měly být součástí rozšířené verze portálu. Asi ideální by bylo, pokud by byly pro běžné uživatele skryty a zpřístupnily se až poté co se mapper přes oauth přihlásí ke svému OSM účtu. 

Pak by se mohly objevit nové volby v menu a extra analytické vrstvy.

Návrh možných analytických vrstev můžeme sbírat tady, konkrétní implementaci pak v separátních issues. Nejprve ale bude potřeba vymyslet a naprogramovat obecné rozhraní.

**Comment by jzvc** [comment]
Ja bych to nedelal na priklaseni, kazdy login == zbytecny klacek pod nohy (pochopitelne, je-li rec o RO pristupu). Proste udelat nejaky rozumne "(ne)napadny", respektive spis nerusivy prvek, kterym se k tomu kdokoli dostane. Ja osobne pouzivam desitky ruznych tools na webu, a kdybych se mel vsude prihlasovat ...

**Comment by zbycz** [comment]
Souhlasím, že by v přepínači stačilo přidat "Pro pokročilé".

Nahodil jsem funkční návrh (wireframe) do toho issuečka #36 - tam by bylo i vhodné rozvést jaké všechny analytické vrstvy chceme.
